tests/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.7'
+
+services:
+  dev-pihole:
+    image: pihole/pihole:latest
+    volumes:
+      - ../:/var/www/html/admin:ro
+    ports:
+      - 8000:80
+    environment:
+      WEBPASSWORD: admin
+
+  npm:
+    image: mcr.microsoft.com/playwright:v1.27.0
```

Comment:
using Playwright's official image as the Node/NPM image ensures versions are always compatible - and one less image to download